web_src/css/base.css

```diff
@@ -2383,9 +2383,9 @@ a.ui.label:hover {
 
 .ui.basic.labels .primary.label,
 .ui.ui.ui.basic.primary.label {
-  background: transparent;
-  border-color: var(--color-primary);
-  color: var(--color-primary);
+  background: transparent !important;
+  border-color: var(--color-primary) !important;
+  color: var(--color-primary) !important;
```

Comment:
Please leave a comment about how it works.

Reply:
Added in https://github.com/go-gitea/gitea/pull/23907/commits/a806ea3240496818bb97db6ceee72394449b8ec7